PDF/include/Poco/PDF/hpdf.h

```diff
@@ -756,6 +756,10 @@ HPDF_EXPORT(HPDF_Image)
 HPDF_LoadJpegImageFromFile (HPDF_Doc      pdf,
                             const char    *filename);
 
+HPDF_EXPORT(HPDF_Image)
+HPDF_LoadBMPImageFromFile(HPDF_Doc      pdf,
+	const char    *filename);
+
```

Comment:
this should go somewhere else, because we want to keep Haru files as a simple drop-in (otherwise, next time we update, this will get overwritten). 

so, options are:
1) move this function to Poco::PDF code (probably a new Util.h/cpp)
2) try to get it accepted upstream, in Haru library

Reply:
Yes you're right, probably we can try with Poco::PDF... Or do you think I can fork somehow Haru lib and propose a pull request?

Reply:
probably best to separate it for now in our own pdf util. libharu maintenance is intermittent, development stopped for a long time, then some work was done, now it seems orphaned again.

but if you want to send them pull, here's the repo: https://github.com/libharu/libharu

Reply:
ok, I'll do the changes as soon as I can. I don't understand the test failing in windows 32bit...is not dependent at all from PDF because it's seems to be in Foundation

Reply:
done, have a look if can be ok

Reply:
can you please rename BmpUtil to simply Util - that way we can add in there anything else in the future, rather than proliferating various util files.

Reply:
otherwise, it's fine